dev/spec-setup.js

```diff
@@ -1,10 +1,15 @@
 /* eslint-disable global-require, no-console */
-
 var rowdy = require("rowdy");
 var MochaAdapter = rowdy.adapters.mocha;
 
+var SERVER_HOST = "127.0.0.1";
+var SERVER_PORT = "3000";
+global.TEST_FUNC_BASE_DIR = process.env.TEST_FUNC_BASE_DIR || "";
+global.TEST_FUNC_BASE_URL = process.env.TEST_FUNC_BASE_URL || "http://" + SERVER_HOST + ":" + SERVER_PORT + global.TEST_FUNC_BASE_DIR;
+
 var config = require("rowdy/config");
 config.options.driverLib = "webdriverio";
+config.settings.local.default.remote = { baseUrl: global.TEST_FUNC_BASE_URL };
```

Comment:
I think I do prefer it coded here as a 'default' so that downstream npm scripts and travisci can specify `ROWDY_OPTIONS` without being forced to specify that baseUrl.

E.g., if I want to just enable rowdy logging or just change the browser, ideally I wouldn't have to pass in that baseUrl.